src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -79,6 +79,32 @@ public void EmitDebugInfoForSourceTextWithoutEncoding2()
 </symbols>", options: PdbValidationOptions.ExcludeMethods);
         }
 
+        [Fact]
+        public void EmitDebugInfoForSynthesizedSyntaxTree()
```

Comment:
Is this test related to this PR?

Reply:
Yes, this is testing that the documents for syntax trees synthesized for syntax nodes won't have checksum in the PDB.